<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerExtensionsManagerWidget</class>
 <widget class="QWidget" name="qSlicerExtensionsManagerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Slicer Extensions Manager</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <widget class="QWidget" name="ManageExtensionsTab">
      <attribute name="icon">
       <iconset resource="../qSlicerBaseQTGUI.qrc">
        <normaloff>:/Icons/Extension.png</normaloff>:/Icons/Extension.png</iconset>
      </attribute>
      <attribute name="title">
       <string/>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QStackedWidget" name="ManageExtensionsPager">
         <widget class="qSlicerExtensionsManageWidget" name="ExtensionsManageWidget"/>
         <widget class="qSlicerExtensionsInstallWidget" name="ExtensionsManageBrowser"/>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="InstallExtensionsTab">
      <attribute name="icon">
       <iconset resource="../qSlicerBaseQTGUI.qrc">
        <normaloff>:/Icons/DownloadExtension.png</normaloff>:/Icons/DownloadExtension.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Install Extensions</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="qSlicerExtensionsInstallWidget" name="ExtensionsInstallWidget" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qSlicerExtensionsManageWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerExtensionsManageWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qSlicerExtensionsInstallWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerExtensionsInstallWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qSlicerBaseQTGUI.qrc"/>
 </resources>
 <connections/>
</ui>
